BP/history/provinces/bp_k_dacia.txt

```diff
@@ -1,54 +1,86 @@
+# Notes:
+# ============
+# Iazyges
+# Iazyges became increasingly diffuse by the IVth century and were settled in and around the Tisza/Tisa river,
+# their tradicinal location [1,2,3]. Listed under [4] are later princes which might be accurate but I can't access
+# the text unfortunately. Also note that the Gepids are listed as only occupying the eastern bank of the Tisza river
+# in this map [5]; I suspect the "Huns" lists under Gepid control were actually Iazyges/Roxolani/Sarmatians.
+#
+# My general impression is that the Iazyges were probably absorbed into the Huns and the 
+# Amospados in WtWSMS represents the Iazyges/Roxolani/Sarmatian groups that survived the Hunnic conquests.
+#
+# [1] https://en.wikipedia.org/wiki/Iazyges#Late_history_and_legacy
+# [2] https://en.wikipedia.org/wiki/Iazyges#/media/File:Iazyges-en.png
+# [3] https://en.wikipedia.org/wiki/Iazyges#/media/File:Pannonia03_en.png
+# [4] https://en.wikipedia.org/wiki/Iazyges#List_of_princes ; E. Stein, J.-R. Palanque. "Histoire du Bas-Empire", 213
+# [5] https://upload.wikimedia.org/wikipedia/commons/0/0b/Europe_and_the_Near_East_at_476_AD.png
+# Other map (AD 125) https://upload.wikimedia.org/wikipedia/commons/b/bb/Roman_Empire_125.png showing Iazyges did not migrate from Tisa basin
+# ============
+# Heruli
+# Probably Quadi/Suebi/Sciri; definitely East German/Alan/Sarmatian mix by 476 [1].
+# Heruli probably didn't extend nearly as far as WtWSMS. [2-4] seems to more accurately represent the Heruls being bound
+# by the Carpathians; given the extent of the Przeworsk-Zarubintsy culture and assuming those were Proto-Slavs, reducing
+# the bounds of the Heruls seems justified. So the Heruls are relative newcomers, ruling over an admixture of people in 
+# the Danube basin while having a Herul stronghold in the mountains
```

Comment:
I normally associate the Slavic/Venedi peoples with the Przeworsk/Zarubintsy cultures, which better fit the southern Visutla region than the Heuli; so far, I only really see references to the Heruli as lying north of the Danube/Vienna and south of the Carpathian Mountains. So I subdivided the previous trans-Carpathian Heruli counties between the Sklaveni, Antae, and Venedi instead of creating a fictional holder.

I'm more circumspect in creating completely fictional tribes near historically attested regions than say in the Sahel or Siberia.

Reply:
No, the point was not to create a fictional tribe, although leaders for these tribes are of course largely fictional due to the lack of sources.